src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -4155,6 +4398,23 @@ namespace AirflowNetworkBalanceManager {
 		static bool OneTimeFlag( true );
 		static bool ErrorsFound( false );
 		Real64 GlobalOpenFactor;
+		Real64 ZonePressure1;
```

Comment:
@mjwitte I am not clear to initialize local variables. For example, ZonePressure1 is a local intermittent variable and is a placeholder to story temporary results. I may move these variables within a tighter scope. Could you give me an example?